### PR TITLE
docs: update comment for hideFromUser

### PR DESCRIPTION
**Pull request by wy-luke**
When creating a terminal with the `hideFromUser: true` option, I have concerns about the possibility of the hidden terminals being restored upon the next opening of the workspace.

To verify this, I tested using `vscode.window.terminals.length` and confirmed that it won't happen.

Considering this, I think it might be beneficial to clarify this point in the comment. So I add `And the hidden terminals won't be restored when the workspace is next opened.` to the end of the comment of `hideFromUser`

### CODE REVIEW

**Comment by wy-luke** [comment]
@microsoft-github-policy-service agree

**Review by Tyriar — DISMISSED** [review]

File: src/vs/platform/terminal/common/terminal.ts [hunk 1]
```diff
@@ -565,7 +565,7 @@ export interface IShellLaunchConfig {
 	 * until `Terminal.show` is called. The typical usage for this is when you need to run
 	 * something that may need interactivity but only want to tell the user about it when
 	 * interaction is needed. Note that the terminals will still be exposed to all extensions
-	 * as normal.
+	 * as normal. And the hidden terminals won't be restored when the workspace is next opened.
```
Comment (TylerLeonhardt) [inline src/vs/platform/terminal/common/terminal.ts:568]: I don't really like starting a sentence with "And"

Comment (Tyriar) [reply src/vs/platform/terminal/common/terminal.ts:568]: Oh same, I guess I didn't save that file when committing as it's meant to be the same

**Review by Tyriar — APPROVED** [review]

**Review by aeschli — APPROVED** [review]

File: src/vscode-dts/vscode.d.ts [hunk 2]
```diff
@@ -11737,7 +11737,7 @@ declare module 'vscode' {
 		 * until `Terminal.show` is called. The typical usage for this is when you need to run
 		 * something that may need interactivity but only want to tell the user about it when
 		 * interaction is needed. Note that the terminals will still be exposed to all extensions
-		 * as normal. And the hidden terminals won't be restored when the workspace is next opened.
+		 * as normal and they will remain hidden when the workspace is reloaded.
```
Comment (wy-luke) [inline src/vscode-dts/vscode.d.ts:11740]: @Tyriar Sorry for the late comment.

Honestly, I don't think your changes have the same meaning as mine. These hidden terminals will not be restored at all, rather being restored and remain hidden. When I had 2 terminals opened, 1 hidden and 1 not, there would be only 1 terminal opened when the workspace is reloaded.

I used `vscode.window.terminals.length` to verify it, which is exactly why I want to add this comment, telling developers these hidden terminals that cannot be closed manually by users won't be restored, no need to worry about that there would be tons of hidden terminal when  the workspace is reloaded.

I'm not sure if I expressed it correctly. Looking forward for your reply. Thanks.

File: src/vs/platform/terminal/common/terminal.ts (hunk 1 above)
Comment (wy-luke) [reply src/vs/platform/terminal/common/terminal.ts]: Sorry about that. I don't know this at all.

File: src/vscode-dts/vscode.d.ts (hunk 2 above)
Comment (Tyriar) [reply src/vscode-dts/vscode.d.ts:11740]: Thanks a bunch for following up and sorry about the delayed response 😅 

I brought back your correct description in https://github.com/microsoft/vscode/pull/207016

**Pull request merged**